pkg/nuctl/command/export.go

```diff
@@ -190,6 +194,7 @@ func (e *exportFunctionCommandeer) renderFunctionConfig(functions []platform.Fun
 type exportProjectCommandeer struct {
 	*exportCommandeer
 	getProjectsOptions platform.GetProjectsOptions
+	withImage          bool
```

Comment:
if it's a common flag, perhaps move it to the common `exportCommandeer`?

Reply:
@TomerShor great idea! done